torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -333,19 +333,20 @@ def _affine_bounding_box_xyxy(
     dtype = bounding_box.dtype if torch.is_floating_point(bounding_box) else torch.float32
     device = bounding_box.device
 
-    affine_matrix = torch.tensor(
-        _get_inverse_affine_matrix(center, angle, translate, scale, shear, inverted=False),
+    affine_vector = _get_inverse_affine_matrix(center, angle, translate, scale, shear, inverted=False)
```

Comment:
Are we sure the right value for this is `inverted=False`? If that's the case, we will need another call with `inverted=True` for the image.

Reply:
It seems the tests are failing and it's probably related to the [inverted](https://github.com/pytorch/vision/actions/runs/3196650769/jobs/5218862358).